tests/utilities/test_enums.py

```diff
@@ -0,0 +1,8 @@
+from pytorch_lightning.utilities import DeviceType
+
+
+def test_consistency():
+    assert DeviceType.TPU not in ('GPU', 'CPU')
+    assert DeviceType.TPU in ('TPU', 'CPU')
+    assert DeviceType.TPU in ('tpu', 'CPU')
+    assert DeviceType.TPU in ('tpu', 'CPU')
```

Comment:
you have different test cases in the PR description. 
double check?

Reply:
```suggestion
    assert DeviceType.TPU not in {'GPU', 'CPU'}
    assert DeviceType.TPU in {'TPU', 'CPU'}
    assert DeviceType.TPU in {'tpu', 'CPU'}
```